backend/views/data.py

```diff
@@ -0,0 +1,29 @@
+from flask import jsonify, request
```

Comment:
Should these be called views or controllers as per MVC architecture?

Reply:
I apologize, I'm not familiar with MVC. Could you please explain more about this?

IMO, this is a model of data for each keyword.

Reply:
@nuugut These are your api endpoints which recieve the request., they should controllers I suppose. You just need to rename the folder views to controllers I guess.

Reply:
Fixed